datadog_checks_base/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - datadog_checks_base
 
+## 8.3.0 / 2019-06-01
+
+* [Added] Use Kube service account bearer token for authentication. See [#3829](https://github.com/DataDog/integrations-core/pull/3829).
+* [Added] Make PDHCheck use PDHBaseCHeck. See [#3818](https://github.com/DataDog/integrations-core/pull/3818).
```

Comment:
Is this and the line above it related to checks_base? If so the wording here should probably be updated.

Reply:
Same with the line above this one :)